systest/scripts/cleanup_tlc_session.sh

```diff
@@ -18,4 +18,5 @@
 set -x
 
 # We only need to cleanup the session, we didn't use barbican-enabled TLC file
```

Comment:
Fix the comment to match mitaka/master edition of this file:
"# Teardown the barbican worker and cleanup the TLC session"